src/js/yeldaWebchatInjector.js

```diff
@@ -440,7 +447,8 @@ class YeldaChat {
     // If parentContainerId presents and valid one, set parentContainer
     if(data.parentContainerId && document.getElementById(data.parentContainerId)) {
       this.parentContainer = document.getElementById(data.parentContainerId)
-      data.canBeClosed = false
+      // Don't overwrite canBeClosed if it has been explicitly set in the webchat config
+      data.canBeClosed = !data.hasOwnProperty('canBeClosed') ? false : data.canBeClosed
```

Comment:
The issue is that on the demo page, the webchat has a parentContainerId because we don't want it to be at the bottom right of the page but in the div with the preview mobile or desktop background 
And because it is a preview of how it would render on the website of the company, we should also be able to close the webchat too
So to achieve that, parentContainerId should not exclude canBeClosed anymore

Reply:
ok
Then please update documentation in all the needed places (including yelda repo) to explain it